schemas/descriptors/deprecated/descriptorDeprecated.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/descriptors/descriptorDeprecated",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Deprecated Field Descriptor",
+  "meta:extensible": false,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/common/descriptors/schemadescriptor#/definitions/descriptor"
+  ],
+  "meta:abstract": false,
+  "type": "object",
+  "description": "Used to mark a standard field as deprecated for the specific ImsOrg.",
```

Comment:
Please change it to.
"Used to mark one or more standard fields as deprecated for the specific ImsOrg.

Reply:
Done